dist/js/tether.js

```diff
@@ -1177,11 +1177,14 @@ var TetherClass = (function (_Evented) {
         }
       };
 
+      var hasOptimizations = typeof this.options.optimizations !== 'undefined';
+      var positionFixed = hasOptimizations ? this.options.optimizations.positionFixed : true;
```

Comment:
As people can already be using the optimizations object for other things, I think the right way to write this is to explicitly check if `positionFixed === false`, and allow it otherwise.  The current method will disable it unless the user specifies every possible key inside `optimizations`.